index.html

```diff
@@ -49,10 +73,208 @@ <h2 class="CONNECT"><a href="#">LET'S CONNECT</a> </h2>
+          
+          
+                  <div class="tonic">
+                      <div><h1>Tonic</h1></div>
+                      <div class="flex1">
+                          <div>Caponi</div>
+                          <div class="round"></div>
+                          <div>back End</div>
+                          <div class="round"></div>
+                          <div>2015</div>
+                      </div>
+                      <div>
+                          <p>
+                              A dayly selection of privatelypersonalized reads:no accounts <br>
+                              or sign-ups required.
+                          </p>
+                      </div>
+                      <div >
+                          <ul class="languages">
+                          <li class="language">HTML</li>
+                          <li class="language">CSS</li>
+                          <li class="language">javascript</li>
+                          </ul>
+                      </div>
+                      <div><button class="see-project" type="button">See project</button></div>
+                  </div>
+                    </div>
+          
+          
+                    <div class="Works">
+                      <div class="img-1"><img class="fontimg"  width="100%"  height="100%" src="Snapshoot4.png" alt=""></div>
+              
+              
+                      <div class="tonic">
+                          <div><h1>Muti-Post <br>Stories</h1></div>
+      
+                          <div class="flex1">
+                              <div>Caponi</div>
+                              <div class="round"></div>
+                              <div>back End</div>
+                              <div class="round"></div>
+                              <div>2015</div>
+                          </div>
+      
+                          <div>
+                              <p>
+                                  A dayly selection of privatelypersonalized reads:no accounts <br>
+                                  or sign-ups required.
+                              </p>
+                          </div>
+      
+                          <div >
+                              <ul class="languages">
+                              <li class="language">HTML</li>
+                              <li class="language">CSS</li>
+                              <li class="language">javascript</li>
+                              </ul>
+                          </div>
+      
+                          <div ><button class="see-project" type="button">See project</button></div>
+                      </div>
+                        </div>
+                   
+          
+          
+           <div class="about-me" >
+          
+             <div class="first-about">
+              <div >
+                  <h1>About<br>Myself</h1>
```

Comment:
- In order to meet the sematic HTML rules kindly avoind using h1 more than once on the same page :recycle: